app/management/club_view.php

```diff
@@ -1,10 +1,11 @@
 <?php
 managementPage("View club");
 $slug = Router::getParameter("slug", pattern: '/[\w-]+/');
-$s = ClubManagementService::fromSlug($slug) ?? force_404("club not found");
+DB::setupForClub($slug);
+$s = ClubManagementService::create($slug) ?? force_404("club not found");
 $club = $s->getClub();
 
-$backupService = new BackupService(dbPath: $s->db->sqlitePath);
+$backupService = new BackupService(dbPath: DB::getInstance()->sqlitePath);
```

Comment:
You can revert this change, it is not needed

Reply:
Done

Reply:
Awesome I will resolve then